app/src/main/res/layout/fragment_mentorship_tasks.xml

```diff
@@ -29,21 +28,24 @@
                 android:id="@+id/rvTasks"
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
-                android:layout_marginLeft="16dp"
-                app:layout_constraintTop_toBottomOf="@id/tvTodo"
-                app:layout_constraintVertical_weight="0.5" />
+                android:layout_marginStart="16dp"
```

Comment:
@anesabml instead of using hardcoded values, please use `margin_normal` value from `dimens.xml` which also measures to 16dp